python/tvm/dlight/benchmark/utils.py

```diff
@@ -101,7 +106,11 @@ def populuate_input_shape(
     return results
 
 
-def default_dym_var_sample_func(dym_var_dict: Dict[str, str]) -> Dict[str, int]:
+def default_dym_var_sample_func(
+    dym_var_dict: Dict[str, str],
+    sample_idx: int,  # pylint: disable=unused-argument
```

Comment:
Seems like `sample_idx` is not used?

Reply:
Yes, this default sample function doesn't use `sample_idx` but it could be very useful for other use cases. For example, we may want to use `n=2^sample_idx` for the samplings.

Reply:
Ah, I see. then it would be nice to give the example in the comments for new users.